src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,103 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private int size;
+    private int threshold;
```

Comment:
This field is redundant and can be calculated from table and load factor, let's remove it

Reply:
which one? are you talking about "threshold" or "size" or both? I'm not agree, why do we have to create a new field every time? I need more explanations.